src/main/java/at/hannibal2/skyhanni/features/inventory/MaxPurseItems.kt

```diff
@@ -0,0 +1,95 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.PurseAPI
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.features.bazaar.BazaarApi
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.client.Minecraft
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class MaxPurseItems {
+
+    private val orderPattern by RepoPattern.pattern(
+        "inventory.maxpurse.order",
+        ".*§6(?<coins>[\\d.,]+) coins §7each.*"
+    )
+    private val instantPattern by RepoPattern.pattern(
+        "inventory.maxpurse.instant",
+        ".*Price per unit: §6(?<coins>[\\d.,]+) coins.*"
+    )
+    private val createOrderPattern by RepoPattern.pattern(
+        "inventory.maxpurse.createorder",
+        "§aCreate Buy Order"
+    )
+    private val createInstantPattern by RepoPattern.pattern(
+        "inventory.maxpurse.createinstant",
+        "§aBuy Instantly"
+    )
```

Comment:
```suggestion
    private val patternGroup by RepoPattern.group("inventory.maxpurse")
    private val orderPattern by patternGroup.pattern(
        "order",
        ".*§6(?<coins>[\\d.,]+) coins §7each.*"
    )
    private val instantPattern by patternGroup.pattern(
        "instant",
        ".*Price per unit: §6(?<coins>[\\d.,]+) coins.*"
    )
    private val createOrderPattern by patternGroup.pattern(
        "createorder",
        "§aCreate Buy Order"
    )
    private val createInstantPattern by patternGroup.pattern(
        "createinstant",
        "§aBuy Instantly"
    )
    
```

I did make this change on the website so this might not 100% work so best to test it as well but should be using a pattern group